### PR TITLE
Fix/user name in wp overview

### PR DESCRIPTION
**Pull request by manwithtwowatches**
https://www.openproject.org/work_packages/13652

### CODE REVIEW

File: public/templates/work_packages.list.details.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h2 ng-bind="workPackage.props.subject"></h2>
     <i class="star icon-star1"/>
     <a href>#{{ workPackage.props.id }}</a>
     <span ng-bind="I18n.t('js.label_added_by')"/>
-    <a href ng-bind="workPackage.props.authorName"/>.
+    <a ng-href="/users/{{ author.props.id }}" ng-bind="author.props.name"/>.
```
Comment (myabc) [inline public/templates/work_packages.list.details.html]: @manwithtwowatches shouldn't this use `PathHelper`?

Comment (manwithtwowatches) [inline public/templates/work_packages.list.details.html]: yup. this was from before the PathHelper refactor. i'll update it now.

**Comment by tilt** [comment]
:+1: (apart from what Alex commented on)

I'd suggest adding a "userLink" directive for these links that gets the user-representing JSON

**Comment by tilt** [comment]
:+1: (waiting for Travis)

**Pull request merged**